apps/web-giddh/src/app/vat-report/obligations/obligations.component.ts

```diff
@@ -308,6 +308,9 @@ export class ObligationsComponent implements OnInit, OnDestroy {
                     label: tax,
                     value: tax
                 }));
+                if (this.taxesList.length === 1) {
```

Comment:
```suggestion
                if (this.taxesList?.length === 1) {
```

Reply:
if (response?.body?.length) {
             
}
this condition already check .

Sir this is Not Required